src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const minDiscont=3;
+  const baseDiscont=7;
+  const baseCost=40*days;
```

Comment:
Consider defining `40` as a constant, such as `COST_PER_DAY`, to improve readability and maintainability.